server/src/main/java/org/elasticsearch/action/admin/indices/resolve/ResolveClusterActionRequest.java

```diff
@@ -51,6 +52,8 @@ public class ResolveClusterActionRequest extends ActionRequest implements Indice
      */
     private boolean localIndicesRequested = false;
     private IndicesOptions indicesOptions;
+    // We mark this field as transient because it's only needed on the coordinator and does not need to be serialized.
+    private transient TimeValue timeout;
```

Comment:
`transient` means "not serialized by Java serialization" but we don't use Java serialization at all, so this is confusing. Please remove the `transient`.

Also it's pretty trappy to have fields that don't get copied over the wire, I think we should only be doing that when there's some very good reason. But I don't see a good reason here, it's only a few bytes. Let's stick to the default behaviour of copying everything.